antlr/src/main/antlr/FSH.g4

```diff
@@ -156,7 +156,7 @@ UNIT:               '\'' (~[\\'])* '\'';
 CODE:               SEQUENCE? '#' (SEQUENCE | CONCEPT_STRING);
 
 
-CONCEPT_STRING:      '"' (~[ \t\r\n\f\\"] | '\\"' | '\\\\')+ (WS (~[ \t\r\n\f\\"] | '\\"' | '\\\\')+)* '"';
+CONCEPT_STRING:      '"' (~[ \t\r\n\f\u00A0\\"] | '\\"' | '\\\\')+ (WS (~[ \t\r\n\f\u00A0\\"] | '\\"' | '\\\\')+)* '"';
```

Comment:
Since we have the `WS` fragment, can this be simplified to:
```
CONCEPT_STRING:      '"' (~(WS|[\\"]) | '\\"' | '\\\\')+ (WS (~(WS|[\\"]) | '\\"' | '\\\\')+)* '"';
```

Reply:
I'm actually not sure -- so that's a real question: _can_ it be simplified to that?

Reply:
Unfortunately, no: it seems that antlr won't allow the negation of the `WS` fragment like that, even though that fragment represents a single character from a set.